maggy/core/experiment_driver/driver.py

```diff
@@ -139,6 +139,7 @@ def run_experiment(self, train_fn: Callable) -> dict:
             # Sparkmagic hb poll intervall is 5 seconds, therefore wait 6 seconds.
             time.sleep(6)
             self.stop()
+            time.sleep(10)
```

Comment:
can you explain why this is needed?

Reply:
I had the issues that the second lagom.experiment function wasn't working. This solved the issue. I think the issue was that the second experiment was called while the first experiments driver was still up.

Reply:
yes but I want to know why 10 seconds? is this always going to work. it's bad practice to just put a sleep and hope it fixes it for all cases. we should understand what's going on. this looks like you are treating only the symptom but didn't fix the cause. also if it didn't happen before, there must be a change that causes it...